force-app/main/default/classes/FSL/WorkOrderImportModel.cls

```diff
@@ -0,0 +1,189 @@
+                return 'Caller Email';
+            return null;
+        }
+    }
+
+    public class WorkOrderLineItemExcel {
+        @AuraEnabled
+        public String FakeId {
+            get; set;
+        }
+        @AuraEnabled
+        public String Category {
+            get; set;
+        }
+        @AuraEnabled
+        public String SubCategory {
+            get; set;
+        }
+        @AuraEnabled
+        public String Issue {
+            get; set;
+        }
+        @AuraEnabled
+        public String ProblemDescription {
+            get; set;
+        }
+        @AuraEnabled
+        public String YardiWorkOrderNumber {
+            get; set;
+        }
+        public String validate() {
+            if (Category == null)
+                return 'Category';
+            if (SubCategory == null)
+                return 'SubCategory';
+            if (Issue == null)
+                return 'Issue';
+            if (YardiWorkOrderNumber == null)
+                return 'Yardi Work Order Number';
+            return null;
+        }
+    }
+
+    public class ServiceAppointmentExcel {
+        @AuraEnabled
+        public String FakeId {
+            get;
+
+            set;
+        }
+        @AuraEnabled
+        public String VendorId {
+            get; set;
+        }
+        @AuraEnabled
+        public String EmployeeCode {
+            get; set;
+        }
+        @AuraEnabled
+        public String Status {
+            get; set;
+        }
+        @AuraEnabled
+        public String ScheduledStartDate {
+            get; set;
+        }
+        @AuraEnabled
+        public String ScheduledFinishDate {
+            get; set;
+        }
+        @AuraEnabled
+        public String ScheduledStartTime {
+            get; set;
+        }
+        @AuraEnabled
+        public String ScheduledFinishTime {
+            get; set;
+        }
+        public String validate() {
+            if (VendorId == null)
+                return 'Vendor ID';
+            if (EmployeeCode == null)
+                return 'Employee Code';
+            if (Status == null)
+                return 'Status';
+            if (ScheduledStartDate == null)
+                return 'Scheduled Start Date';
+            if (ScheduledFinishDate == null)
+                return 'Scheduled Finish Date';
+            if (ScheduledStartTime == null)
+                return 'Scheduled Start Time';
+            if (ScheduledFinishTime == null)
+                return 'Scheduled Finish Time';
+            return null;
+        }
+    }
+
+    @AuraEnabled
+    public List<WorkOrderExcel> WorkOrders {
+        get; set;
+    }
+    @AuraEnabled
+    public List<WorkOrderLineItemExcel> WorkOrderLineItems {
+        get; set;
+    }
+    @AuraEnabled
+    public List<ServiceAppointmentExcel> ServiceAppointments {
+        get; set;
+    }
+    public String validate() {
+        if (WorkOrders == null || WorkOrders.size() < 1) {
```

Comment:
it's better to use `.isEmpty()`.
` if (WorkOrders == null || WorkOrders.isEmpty()) {}`